tests/release/e2e-test-releaseplan-owner-ref.go

```diff
@@ -42,8 +42,16 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-2469]test-releaseplan-owner-ref-a
 		It("verifies that the ReleasePlan has an owner reference for the application", func() {
 			releasePlan, err = fw.AsKubeAdmin.ReleaseController.GetReleasePlan(sourceReleasePlanName, devNamespace)
 			Expect(err).NotTo(HaveOccurred())
-			ownerRef := releasePlan.OwnerReferences[0]
-			Expect(ownerRef.Name).To(Equal(applicationNameDefault))
+			Eventually(func() error {
+				if len(releasePlan.OwnerReferences) < 1 {
+					return fmt.Errorf("OwnerReferences is not created in ReleasePlan %s yet", releasePlan.Name)
```

Comment:
```suggestion
					return fmt.Errorf("OwnerReference not updated yet for ReleasePlan %s yet", releasePlan.Name)
```
You can split it in two though. First check to see if there any and another one to check if there's only one. Up to you.

Reply:
I guess last word **yet** in your suggestion should be removed ? 
like below : 

return fmt.Errorf("OwnerReference not updated yet for ReleasePlan %s ", releasePlan.Name)